barretenberg/cpp/src/barretenberg/vm2/simulation/alu.test.cpp

```diff
@@ -64,5 +73,46 @@ TEST(AvmSimulationAluTest, NegativeAddTag)
         ElementsAre(AluEvent{ AluOperation::ADD, a, b, MemoryValue::from_tag(a.get_tag(), 0), AluError::TAG_ERROR }));
 }
 
+TEST(AvmSimulationAluTest, LT)
+{
+    EventEmitter<AluEvent> alu_event_emitter;
+    StrictMock<MockRangeCheck> range_check;
+    StrictMock<MockFieldGreaterThan> field_gt;
+    Alu alu(range_check, field_gt, alu_event_emitter);
+
+    auto a = MemoryValue::from<uint32_t>(1);
+    auto b = MemoryValue::from<uint32_t>(2);
+
+    EXPECT_CALL(range_check, assert_range(0, /*num_bits=*/32));
+
+    auto c = alu.lt(a, b);
+
+    EXPECT_EQ(c, MemoryValue::from<uint1_t>(1));
+
+    auto events = alu_event_emitter.dump_events();
+    EXPECT_THAT(events, ElementsAre(AluEvent{ AluOperation::LT, a, b, c, {} }));
```

Comment:
nit: prefer to specify the names for the struct, eg.
```
AluEvent{
.a = a,
.b = b,
.c = c
}
```
relying on order is dangerous

You can probably ignore the `{}` if it's the default value (but I'm not against doing it explicitly).